prow-configurations/install-prow.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -o errexit
+
+kubectl create clusterrolebinding cluster-admin-binding \
+  --clusterrole cluster-admin --user $(gcloud config get-value account)
+
+# Deploy NGINX Ingress Controller
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/mandatory.yaml
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/provider/cloud-generic.yaml
+
+# Follow the installation steps in https://github.com/kubernetes/test-infra/blob/master/prow/getting_started.md#how-to-turn-up-a-new-cluster
+
+## Create an HMAC token
+hmac_token="$(openssl rand -hex 20)"
+echo "hmac-token: $hmac_token"
+
+if [ "$OAUTH" == "" ]; then
+    echo -n "Enter OAuth2 token that has read and write access to the bot account, followed by [ENTER]: "
+    read oauth_token
+else
+    oauth_token="$OAUTH"
+fi
+
+kubectl create secret generic hmac-token --from-literal=hmac=$hmac_token
+kubectl create secret generic oauth-token --from-literal=oauth=$oauth_token
+
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/test-infra/a202e595a33ac92ab503f913f2d710efabd3de21/prow/cluster/starter.yaml
```

Comment:
Let's install it to a different namespace than the default. Also, can we configure it to run prow jobs in separate namespace than prow components?

Reply:
It looks problematic to install Prow on a namespace because of this issue https://github.com/kubernetes/test-infra/issues/9394.